### DISCUSSION TITLE
主页配置

### DISCUSSION THREAD

**Discussion by PanZiwei** [wangchucheng/hugo-eureka#47] [Ideas]
你好，

1. 菜单栏能否添加分类(category)或者标签(tags)一栏？
效果举例：
https://themes.gohugo.io//theme/hugo-theme-meme/categories/
https://hugoloveit.com/categories/
https://themes.gohugo.io//theme/hugo-theme-meme/tags/
https://hugoloveit.com/tags/ 

2. 日志展示时能否仅显示摘要部分（而不是显示前XX字）？目前日志显示的是创作时间，阅读时长和分类，怎么添加其他属性呢（比如最后修改时间）？

3. 是否考虑支持Waline评论系统/Valine评论系统/utterances（基于 GitHub Issues 的评论系统）？
Waline官网：https://waline.js.org/
Valine官网：https://valine.js.org/index.html
utterances: https://github.com/utterance/utterances
别人写的静态博客系统比较：https://eallion.com/comments/

4. 是否考虑添加搜索功能？

5. 添加一个Github issue，以汇总使用该主题的用户及博客链接？

**Comment by wangchucheng** [comment]
> 1. 菜单栏能否添加分类(category)或者标签(tags)一栏？

可以的，菜单栏的更详细的配置方式可参考：https://gohugo.io/content-management/menus/

> 2\. 日志展示时能否仅显示摘要部分（而不是显示前XX字）？

可以的，你可以设置标签于你希望的位置，该标签前的内容会成为摘要。你也可以在Front Matter中设置description的值。

> 3\. 是否考虑支持Waline评论系统/Valine评论系统/utterances（基于 GitHub Issues 的评论系统）？

目前有这个计划但是因为已经支持了两个评论系统所以优先级不高，欢迎提出对该功能的pr！

> 4\. 是否考虑添加搜索功能？

是的，这是一个优先级较高的功能，预计至少将引入一种搜索。

> 5\. 添加一个Github issue，以汇总使用该主题的用户及博客链接？

是一个可以考虑的方式，我对方式等进行考虑，谢谢！
